package.json

```diff
@@ -71,6 +71,6 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "d3-dsv": "^1.0.3",
-    "d3-request": "^1.0.2"
```

Comment:
A fresh install yields v 1.0.6 which breaks the build.

Reply:
Yeah I had the same problem on another PR. Thank you for the good version :D